l2geth/rollup/sync_service.go

```diff
@@ -680,13 +577,6 @@ func (s *SyncService) ApplyTransaction(tx *types.Transaction) error {
 		return fmt.Errorf("invalid transaction: %w", err)
 	}
 
-	if tx.L1Timestamp() == 0 {
```

Comment:
This needs to be added back in

Reply:
Note: this does not need to be added back in, this logic is handled in `SyncService.applyTransactionToTip`